### DISCUSSION TITLE
How cloud I run the dev vscode in the github codespace

### DISCUSSION THREAD

**Discussion by ckmilse** [microsoft/vscode-discussions#613] [Extension Development QnA]
I fork the "microsoft/vscode", and create one codespace; install the dependencies;

1. I run the "run and build " task, then run the command `./scripts/code-server.sh --launch`
2. but there is no "9888 port" in the portList Tab

![截屏2023-05-23 14 47 45](https://github.com/microsoft/vscode-discussions/assets/10125008/85a57076-0056-4d2b-a1a8-b1fb8c4abe5e)

**Comment by olguzzar** [comment]
This forum is specifically for extension development Q&A, so I'm going to close this discussion. If you haven't already, check out the [How to Contribute guide](https://github.com/microsoft/vscode/wiki/How-to-Contribute#development-container) for the microsoft/vscode repo.
